spec.bs

```diff
@@ -432,6 +436,23 @@ To <dfn>register an import map</dfn> given a [=string=] or a null |source text|,
   1. Return null.
 </div>
 
+<div algorithm>
+  To <dfn lt="validate the module script URL|validate a module script URL">validate a module script URL</dfn>, given a [=URL=] |url|, an [=environment settings object=] |settings object|, and a [=URL=] |base URL|:
+
+  1. <div class="note">Optionally, we can apply context-dependent access restrictions based on |base URL|. For example, Chromium might implement the followings, which aren't observable by users.
+         1. If |url|'s [=url/scheme=] is "`std-internal`" and |base URL|'s [=url/scheme=] is "`std-internal`", then return. This allows the use of "`std-internal`" within built-in modules, and rejects "`std-internal`" from user-provided scripts.
+         1. If |url|'s [=url/scheme=] is "`std`", |url| represents an access-restricted built-in module, and |base URL|'s [=url/scheme=] is not "`std-internal`", then throw a {{TypeError}}. This will be used to restrict access to getOriginals only within built-in modules.
```

Comment:
The intention for our initial internal get-originals implementation is to use (1) here, not (2). I think (2) is a bit confusing and I would probably remove it.

Reply:
Sounds simpler.
So you mean get-originals have `std-internal:` URLs but not `std:` URLs, right?
In this way, this section just says "Chromium-internal things can't be observed from outside", which looks trivial.
(Anyway I'll leave this section, to guide implementation)

Reply:
Removed (2).